Samples/PowerShell/functions/func_ModernDashboards.ps1

```diff
@@ -115,37 +255,52 @@ function Export-ModernDashboard {
     }
     Process {
         ForEach ( $d in $DashboardId ) {
-            $DashboardText = Invoke-SwisVerb -SwisConnection $SwisConnection -EntityName Orion.Dashboards.Instances -Verb Export -Arguments $d
+            $DashboardText = Invoke-SwisVerb -SwisConnection $SwisConnection -EntityName Orion.Dashboards.Instances -Verb Export -Arguments $d -ErrorAction SilentlyContinue
```

Comment:
Why `SilentlyContinue`?

Reply:
In the event that it errors out.  It could be converted to a `try/catch/finally` but I elected for the easy way out today.